### PR TITLE
New package: strongswan-5.3.4

### PR DESCRIPTION
**Pull request by Vaelatern**

### CODE REVIEW

File: srcpkgs/strongswan/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'xl2tpd'
```
Comment (Gottox) [inline srcpkgs/strongswan/template]: change this to strongswan

**Comment by Gottox** [comment]
Good so far. I think this will need an sv file too.

**Comment by xtraeme** [comment]
I really doubt this is ok... shared libraries without development packages.

**Comment by xtraeme** [comment]
Adding just shared libraries in `makedepends` without `development packages` is also completely wrong.

**Comment by Vaelatern** [comment]
Can you explain what needs to be done for `development packages`? Just vmove the files as specified in the manual?

Tried looking at the systemd file, but can't find the executables that it calls. Was hoping for some help in writing the sv file.

**Comment by Gottox** [comment]
See: https://github.com/voidlinux/void-packages/blob/master/Manual.md#libraries

**Comment by Vaelatern** [comment]
I believe the provided library files belong in the main package, not a -devel package.

**Comment by pullmoll** [comment]
The unversioned `/usr/lib/*.so` files and any static `/usr/lib/*.a` libraries belong into -devel, as well as header files, pkgconfig files, cmake helpers etc.

The versioned `/usr/lib/*.so.*` files should stay in a main package, or in a `lib<packge>` subpackage which main depends upon.

**Comment by Gottox** [comment]
Nevertheless, I believe that strongswan is not a Library at all. Rather remove those files from common/shlibs.

**Comment by Vaelatern** [comment]
Went with Gottox's thought, as I find it hard to think strongswan is a library. I simply got warnings during `./xbps-src install strongswan` and blindly tried to fix them.

**Comment by pullmoll** [comment]
True. I see no other distribution defining a strongswan-dev(el) package.

**Comment by xtraeme** [comment]
- Please disable static libraries, they are useless if you can't use them i.e `--disable-static` in `configure_args`.
- Set `build_pie=yes` to build it with hardening features (PIE, relro, etc).

**Comment by Gottox** [comment]
Please update to 5.3.4. See https://www.strongswan.org/blog/2015/11/16/strongswan-vulnerability-%28cve-2015-8023%29.html

**Comment by Vaelatern** [comment]
`build_pie` added then removed when it became default.

**Pull request merged**